docs/application/vstools/tools/profiler-user-manual.md

```diff
@@ -0,0 +1,263 @@
+      - To open **Tizen** options select in the menu **Tools > Options > Tizen > Tools**.
+      - Turn on option `Use new debugger when available`.
+      - Turn on option `Use Live Profiler on debugging`.
+
+      ![Tizen options](media/tizen_options_live_profiler.png)
+
+## Profiling Progress window
+  The window shows CPU Utilization and Memory consumption by application in the real-time mode.
+
+  ![Profiling Progress window](media/profiling_progress.png)
+
+  - On the top right corner buttons **Pause/Resume** and **Stop** are located. But if the profiling was started in the `Delayed` mode instead **Pause/Resume** button **Start** button will be to start profiling.
+
+  - Depends on the profiling options it is possibly to track JIT and GC events in the real-time mode.
+
+    ![Profiling Progress window: JIT and GC events](media/profiling_progress_jit_gc.png)
+
+  - In the debug mode the window shows debugger events, for example : when an application was paused by breakpoint or debug pause button.
+
+    ![Profiling Progress window: Debugger events](media/profiling_progress_debugger.png)
+  
+  - To pause profiling click **Pause** button.
+
+  - To resume profiling click **Resume** button.
+
+  - To stop profiling click **Stop** button.
+
+  - On the left bottom corner current status is displayed.
+
+## Managing session
+  - To open **Session Explorer** window select in the menu **Tools > Tizen > Profiler > Session Explorer**.
+   
+     ![Open Session](media/open_session.png)
+
+  - **Session Explorer** window contains a list of profiling sessions which have been collected within opened Solution. A list entry which is mapped to the session has the next describing fields:
+    - Date - date and time of profiling session.
+    - Preset - name of the used Profiling preset to create profiling session.
+    - Device - name of the used device for profiling.
+    - Project - name of the profilled project.
+    - Annotation (optional) - user defined Session annotation.
+      
+      ![Session Explorer](media/session_explorer.png)
+
+  - Additional session's information is displayed in **Properties** window (usually it is placed below **Session Explorer** window). The window contains information about specific session's parameters that had been used to configure the profiler for that session. Annotation field can be edited in this window.
+    
+      ![Session Properties](media/session_properties.png)
+
+  - To open session double click on the session's entry.
+  - To manage session click `Right Mouse Button` on the session's entry.
+      
+    ![Session's entry Context menu](media/mouse_right.png)
+
+    The next possible operations with session entry are available:
+    - Open Profile Session
+    - Open Memory Profile Session (availability depends on the session's parameters)
+    - Edit Annotation
+    - Delete
+
+## Analyzing session data
+  Depends on the chosen Profiling preset there are available two types of the session data's viewers:

```

Comment:
Depending on the Profiling preset, following are the types of the session data's viewers: